_posts/2024-11-15-dynamic-tables.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: postv2
+title: "Getting Dynamic with Snowflake"
+description: "Exploring Snowflake's Dynamic Tables feature"
+date: 2024-11-18
+highlight: monokai
+image: /img/dynamic-tables/dt_graph.png
+author: Ben Dang
+tags:
+  [
+    Snowflake,
+    Dynamic Tables,
+    Data
+    ]
+---
+
+
+# Dynamic Tables
+
+After more than a year in preview, Snowflake finally launched Dynamic Tables for general availability in [April 2024](https://docs.snowflake.com/en/release-notes/2024/other/2024-04-29-dynamic-tables). For those of us in data engineering, this is massive as Dynamic Tables brings together scheduling and data ingestion into one setup, all in SQL. This means we can handle batch and streaming data seamlessly without needing complex infrastructure or extra tools. After implementing and monitoring Dynamic Tables for a few months with a client, here’s my take on how they’re reshaping data pipelines.
```

Comment:
Maybe a couple more sentences about what dynamic tables do would be beneficial - an example code how to create a dynamic table and what it ahieves would help to understand them.